packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -105,17 +105,23 @@ export class DocumentStorageService implements IDocumentStorageService {
         /** Entire previous snapshot, not subtree */
         previousFullSnapshot: ISnapshotTree | undefined,
     ): Promise<string> {
-        const entries = await Promise.all(Object.keys(summaryTree.tree).map(async (key) => {
-            const entry = summaryTree.tree[key];
-            const pathHandle = await this.writeSummaryTreeObject(key, entry, previousFullSnapshot);
-            const treeEntry: resources.ICreateTreeEntry = {
-                mode: this.getGitMode(entry),
-                path: encodeURIComponent(key),
-                sha: pathHandle,
-                type: this.getGitType(entry),
-            };
-            return treeEntry;
-        }));
+        const entries = await Promise.all(Object.keys(summaryTree.tree)
+            // Attachment entries are used for garbage collection, which we don't do on r11s, so just ignore them
+            .filter((key) => {
+                const entry = summaryTree.tree[key];
+                return entry.type !== SummaryType.Attachment;
```

Comment:
I assume that in case of r11s, that ID is a sha, and we should be fine writing it out as well (i.e. keep parity with SPO).
I remember we chatted about it many times, but I do not remember why filtering out was considered.  Should we keep all IDs in?